src/api/Gemfile

```diff
@@ -119,6 +119,8 @@ group :test do
   gem 'rails-controller-testing'
   # To generate random data
   gem 'rantly', '>= 1.1.0'
+  # To ensure consistent Git commits
+  gem 'git-cop', git: 'https://github.com/Ana06/git-cop.git', branch: 'travis-ci'
```

Comment:
It's not possible to use gems from git for us, we need to be able to package them.